lib/http/endpoint.js

```diff
@@ -223,6 +223,10 @@ const defaultResultWriter = (result, request, response, next) => {
 // error thrown upstream that is of our own internal format, this handler does
 // the necessary work to translate that error into an HTTP error and send it out.
 const defaultErrorWriter = (error, request, response) => {
+  if (error instanceof URIError) {
+    error = Problem.user.notFound(); // eslint-disable-line no-param-reassign
```

Comment:
What do you think of this alternative to reassigning `error`?

```suggestion
    return defaultErrorWriter(Problem.user.notFound(), request, response);
```